IdentityCore/src/throttling/cache/MSIDLRUCache.m

```diff
@@ -167,12 +167,12 @@ - (BOOL)setObject:(id)cacheRecord
     dispatch_barrier_sync(self.synchronizationQueue, ^{
         if (self.cacheSizeInt <= DEFAULT_CACHE_OFFSET_SIZE)
         {
-            subError = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, @"MSIDLRUCache Error: cache was initialized with size less than 1. Cannot write due to insufficient size.", nil, nil, nil, nil, nil, YES);
+            subError = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, @"MSIDLRUCache Error: cache was initialized with size less than 1. Cannot write due to insufficient size.", nil, nil, nil, nil, nil, NO);
```

Comment:
any reason not log error description?

Reply:
It's from this PR: https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/1064
This one just based on the same source version, wouldn't show this change when that PR merged into dev.